tests/RadixDlt.NetworkGateway.UnitTests/Abstractions/Utilities/ClampDateTimeFormatterTests.cs

```diff
@@ -0,0 +1,91 @@
+ * trustees, representatives, agents, advisors, contractors, or volunteers of the Licensor
+ * shall be liable for any direct or indirect, special, incidental, consequential or other
+ * losses of any kind, in tort, contract or otherwise (including but not limited to loss
+ * of revenue, income or profits, or loss of use or data, or loss of reputation, or loss
+ * of any economic or other opportunity of whatsoever nature or howsoever arising), arising
+ * out of or in connection with (without limitation of any use, misuse, of any ledger system
+ * or use made or its functionality or any performance or operation of any code or protocol
+ * caused by bugs or programming or logic errors or otherwise);
+ *
+ * A. any offer, purchase, holding, use, sale, exchange or transmission of any
+ * cryptographic keys, tokens or assets created, exchanged, stored or arising from any
+ * interaction with the Work;
+ *
+ * B. any failure in a transmission or loss of any token or assets keys or other digital
+ * artefacts due to errors in transmission;
+ *
+ * C. bugs, hacks, logic errors or faults in the Work or any communication;
+ *
+ * D. system software or apparatus including but not limited to losses caused by errors
+ * in holding or transmitting tokens by any third-party;
+ *
+ * E. breaches or failure of security including hacker attacks, loss or disclosure of
+ * password, loss of private key, unauthorised use or misuse of such passwords or keys;
+ *
+ * F. any losses including loss of anticipated savings or other benefits resulting from
+ * use of the Work or any changes to the Work (however implemented).
+ *
+ * You are solely responsible for; testing, validating and evaluation of all operation
+ * logic, functionality, security and appropriateness of using the Work for any commercial
+ * or non-commercial purpose and for any reproduction or redistribution by You of the
+ * Work. You assume all risks associated with Your use of the Work and the exercise of
+ * permissions under this License.
+ */
+
+using FluentAssertions;
+using RadixDlt.NetworkGateway.Abstractions.Utilities;
+using Xunit;
+
+namespace RadixDlt.NetworkGateway.UnitTests.Abstractions.Utilities;
+
+public class ClampDateTimeFormatterTests
+{
+    [Theory]
+    [InlineData(-9223372036854775808L, "1583-01-01T00:00:00Z")]
+    [InlineData(-62135596800L, "1583-01-01T00:00:00Z")]
+    [InlineData(-12212553600L, "1583-01-01T00:00:00Z")]
+    [InlineData(-1L, "1969-12-31T23:59:59Z")]
+    [InlineData(0L, "1970-01-01T00:00:00Z")]
+    [InlineData(1L, "1970-01-01T00:00:01Z")]
+    [InlineData(1713447455L, "2024-04-18T13:37:35Z")]
+    [InlineData(12212553601L, "2357-01-01T00:00:01Z")]
+    [InlineData(253402300799L, "9999-12-31T23:59:59Z")]
+    [InlineData(253402300800L, "9999-12-31T23:59:59Z")]
+    [InlineData(9223372036854775807L, "9999-12-31T23:59:59Z")]
+    public void Tests(long timestamp, string expected)
```

Comment:
Nice test!

Super-minor-optional-suggestion to make it very obvious: 
* `sed/timestamp/unixTimestampSecs`
* `sed/Tests/ScryptoInstantClampingTests`

Reply:
Makes sense, changed.